README.md

```diff
@@ -94,6 +95,12 @@ Login to your Trading 212 account and create an export file (via History > Downl
 
 Login to your XTB account and from the top bar click on "Account history", then "Cash operations". Click the "Export button". Choose the period from which you wish to export your history, select report type "Cash Operations" choose file format "csv" then click "Export Report" button.
 
+### Investimental
```

Comment:
Could you move this up, so it is alphabetically sorted?

Reply:
done